src/seedu/addressbook/Main.java

```diff
@@ -75,11 +76,25 @@ private void exit() {
         ui.showGoodbyeMessage();
         System.exit(0);
     }
+    
+    /**
+     * Throws exception if user deletes storage file. 
+     */
+    private void checkIfStorageFileDeleted() throws FileNotFoundException {
+    	if (!storage.path.toFile().exists()) { // check if storage file exists
+    		throw new FileNotFoundException("User deleted storage file");
+    	}
+    }
 
     /** Reads the user command and executes it, until the user issues the exit command.  */
     private void runCommandLoopUntilExitCommand() {
         Command command;
         do {
+        	try {
+        		checkIfStorageFileDeleted();
+        	} catch (FileNotFoundException fnfe) {
+        		fnfe.printStackTrace();
```

Comment:
This is an ok a place to catch the exception but this is not the way to HANDLE the exception. Just a printStackTrace of the error is not acceptable.

You can do two things here, (1) show a warning to the user and terminate or (2) show a warning, recover and continue.